src/browse.py

```diff
@@ -31,7 +31,10 @@
 import subprocess
 import sys
 import webbrowser
-import urllib2
+try:
+    from urllib2 import unquote
```

Comment:
Isn't the recommendation "try py3 variant, use py2 as fallback"? (I'm not sure, I don't do much py3.)